content/en/docs/instrumentation/js/getting-started/browser.md

```diff
@@ -85,7 +107,7 @@ In the following we will use [parcel](https://parceljs.org/) as web application
 Run
 
 ```shell
-npx parcel index.html
+$ npx parcel index.html
```

Comment:
Don't put a `$` for `shell` environments (which is the proper code-block lang to use when there is no command output):

```suggestion
npx parcel index.html
```